cli/flox/src/commands/generations/mod.rs

```diff
@@ -22,11 +22,11 @@ pub enum GenerationsCommands {
     #[bpaf(command)]
     List(#[bpaf(external(list::list))] list::List),
 
-    /// Show all managed changes to the selected environment
+    /// Print history of the current generation
     #[bpaf(command)]
     History(#[bpaf(external(history::history))] history::History),
 
-    /// Switch to the previously active generation
+    /// Switch to the previous generation
```

Comment:
This implies to be that it will rollback linearly to N-1.

Reply:
Just dropped the change for now and may followup if we come up with a replacement for current